android/guava-tests/test/com/google/common/primitives/UnsignedBytesTest.java

```diff
@@ -242,7 +242,7 @@ private static boolean unsafeComparatorAvailable() {
        * See https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4.2 (steps #11 and
        * #5).
        *
-       * Note that that "first time" might happen in a test other than
+       * Note that "first time" might happen in a test other than
```

Comment:
Not an error, for a similar reason to FinalizableReferenceQueueClassLoaderUnloadingTest:58.